src/components/HomeGrid.js

```diff
@@ -15,9 +15,10 @@ import GridCard from './GridCard.js';
 const HomeGrid = () => (
   <Grid
     columns={{ count: 'fill', size: 'medium' }}
-    gap="medium"
+    gap="large"
     rows="auto"
     margin={{ horizontal: 'xlarge', top: 'small', bottom: '-30px' }}
+    alignSelf="center"
```

Comment:
I think this may be causing the issue

Reply:
Yeah I wasn't sure what the best way to do this was. The way the cards are before they don't scale small enough for mobile view. The way the cards are now in this PR they scale small enough but they look different when the screen is wide. Maybe we can ask shimi and marisa tomorrow